### DISCUSSION TITLE
Icon VDH v9.1.0.44 doesn't show / interact in Firefox 130.0 (64-bits)

### DISCUSSION THREAD

**Discussion by smfukaya** [aclap-dev/video-downloadhelper#1249] [Help and Q&A]
### Your question

I'm not sure if this is a problem with Firefox 130.0 (64-bits) in Brazilian Portuguese over Microsoft Windows 10 Version 22H2 (Build 19045.4842) or it is a problem with VDH v9.1.0.44, but the extension icon is not showing anymore nor in extensions context menu neither when the extension is pinned to the toolbar (and it does nothing when clicked).

I manually disable all other extensions and restarted Firefox, but the extension is still not working.

The only thing that makes it work is restarting Firefox in troubleshooting mode which removes all extensions. Then, I can add it again, but I need to revalidate the license key. And the extension will work only until the next PC restart (yes, I already restored Firefox, but I am syncing extension settings). I did that so many times that my license got blocked.

**Extension button context menu**:
![image](https://github.com/user-attachments/assets/c76e9638-6756-4e28-b3d9-a505a6233315)

**VDH pinned to Firefox toolbar**:
![image](https://github.com/user-attachments/assets/b35ff6d8-147f-48f9-8c6c-4ba542100d64)

**VDH pinned to Firefox toolbar right-clicking**:
![image](https://github.com/user-attachments/assets/644d57d0-7c39-4085-89a7-a96b26fb7a4a)

### Addon & CoApp version and details
```
C:\Program Files\DownloadHelper CoApp>vdhcoapp.exe --info
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\XXXXXXX",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}
```

**Comment by Mattmjs** [comment]
Try removing the extension and install it again and see if it helps or not.

**Comment by smfukaya** [comment]
Thanks @Mattmjs, but after a few tries, the only thing that prevented losing the extension settings was creating a new profile (`about:profiles`) outsite "%APPDATA%\Mozilla\Firefox\Profiles" as described in the solution for this post [Firefox keeps resetting settings after PC restart ](https://www.reddit.com/r/firefox/comments/1e7rm20/comment/lf3onjc/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)
